context/app/static/js/pages/Collections/types.ts

```diff
@@ -0,0 +1,17 @@
+interface Collection {
+  uuid: string;
+  title: string;
+  hubmap_id: string;
+  datasets: { hubmap_id: string }[];
+}
+
+interface CollectionHit {
+  _source: Collection;
+}
```

Comment:
Let's keep this for now, once we start moving away from searchkit in earnest I think this would be a prime case for a reusable generic type:

```
interface Hit<T> {
  _source: T
}
```

Reply:
Agreed, I didn't want to get too far into it here.